src/app/app.component.html

```diff
@@ -23,12 +23,12 @@
 </mat-toolbar>
 <nav mat-tab-nav-bar>
   <a mat-tab-link
-     [routerLink]="'/patternLanguages'"
+     [routerLink]="'/' + patternLanguagesPath"
      routerLinkActive #rla1="routerLinkActive" [active]="rla1.isActive">
     Pattern Languages
   </a>
   <a mat-tab-link
-     [routerLink]="'/patternViews'"
+     [routerLink]="'/' + patternViewsPath"
```

Comment:
Not sure if it is out of scope or not, but may consider to change this for the other six links as well.